app/lib/queues/populatePracticeQueue.js

```diff
@@ -12,16 +12,18 @@ const steps = ['overview', 'facilities', 'services'];
 const numberOfSteps = steps.length;
 let hitsPerWorker;
 let count = 0;
+let retryCount = 0;
```

Comment:
Not sure how this would be achieved, as these value are used throughout the module.
The values are generally for reporting purposes only, i.e. how far the scrape has got through the process, any unintended changes would not affect the running of ETL.